Sources/FluentKit/Query/Builder/QueryBuilder.swift

```diff
@@ -256,4 +293,15 @@ public final class QueryBuilder<Model>
         }
         return done
     }
+
+    private func addTimestamps(triggers: [TimestampTrigger], nested: inout [FieldKey: DatabaseQuery.Value]) {
+        let timestamps = Model().timestamps.filter { triggers.contains($0.trigger) }
+
+        for timestamp in timestamps {
+            let path = timestamp.path.first!
```

Comment:
This will fail if the `@Timestamp` appears in an `@Group`. We should either give a clearer fatalError that that's not supported yet or properly implement this and add a test for it.